src/Resource/Repository.js

```diff
@@ -14,7 +15,7 @@ export default {
    * @returns {Promise<HyralResource>}
    */
   findOne(parameterBag) {
-    return this.find(parameterBag).then(response => response[0]);
+    return this.find(parameterBag).then(response => response.data[0] || {});
```

Comment:
Wellicht beter om null te returnen? Dat is iets meer conform de waarheid en makkelijker checken met een if statement

Reply:
Eens.